<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWeatherForecast</class>
 <widget class="QMainWindow" name="QWeatherForecast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CWeatherForecast</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="switchModeBtn">
        <property name="text">
         <string>实时天气</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cityEdit">
        <property name="placeholderText">
         <string>请输入城市/城市编码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="selectModeWgt" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="provinceComboBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="cityComboBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="countyComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>报告时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="reportTimeLab">
        <property name="text">
         <string>报告时间</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="1,1,3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="liveWeatherCityLab">
            <property name="text">
             <string>城市标签</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="liveTempLab">
            <property name="text">
             <string>°C</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="liveWeatherImgLab">
            <property name="text">
             <string>天气图标</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="liveWeatherLab">
            <property name="text">
             <string>天气</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lab">
              <property name="text">
               <string>风速：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="liveWindPowerLab">
              <property name="text">
               <string>风速</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>风向：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="liveWindDirectionLab">
              <property name="text">
               <string>风向</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>湿度：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="liveHumidityLab">
              <property name="text">
               <string>湿度</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="forecastLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="rightMargin">
           <number>80</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="forecastWeatherImgLab">
            <property name="text">
             <string>天气图标</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="forecastHighTempLab">
            <property name="text">
             <string>°C</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>风向：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="forecastWindDirectionLab">
              <property name="text">
               <string>风向</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="forecastLowTempLab">
            <property name="text">
             <string>°C</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lab_2">
              <property name="text">
               <string>风速：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="forecastWindPowerLab">
              <property name="text">
               <string>风速</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="forecastWeatherLab">
            <property name="text">
             <string>天气</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="forecastWeatherCityLab">
            <property name="text">
             <string>城市标签</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="forecastDateLab">
            <property name="text">
             <string>星期 日期</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="refreshAction">
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
  <action name="cityChangedAction">
   <property name="text">
    <string>切换城市输入方式</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
